Sources/App/configure.swift

```diff
@@ -18,6 +18,10 @@ import Vapor
 
 @discardableResult
 public func configure(_ app: Application) throws -> String {
+    // The bundle is only loaded if /Applications/InjectionIII.app exists on the local development machine.
+    // Requires InjectionIII 4.7.3 or higher to be compaible with Package.swift files.
+    let _ = Bundle(path: "/Applications/InjectionIII.app/Contents/Resources/macOSInjection.bundle")?.load()
```

Comment:
Since this is all for debug config only, it'd probably best to also wrap this in `#if DEBUG`, just to make that part explicit.

Reply:
Or even `#if DEBUG && os(macOS)` actually.

Reply:
I was also thinking we might want to gate it on an environment variable (or similar), too, so that it's possible to turn it off temporarily without uninstalling/moving the app.

Reply:
You can set the INJECTION_DAEMON environment variable to force using the app which if you're not running it will disable injection.

Reply:
Thanks John! I added a comment to let people know about that and it works great for me.

Reply:
Using the App may better for you if you're not using the Xcode editor. If the code doesn't compile you get a change from orange (ready) to yellow (error) of the icon on the menu bar. The other states are not connected (blue) and compiling (compiling),